src/ops/ops.go

```diff
@@ -56,6 +54,7 @@ type ops struct {
 	log       *logrus.Logger
 	logWriter *utils.LogWriter
 	cmdEnv    []string
+	Runner    opsexec.Excecutor
```

Comment:
Any reason not to use a pointer here?

Reply:
If the goal is to "hide" the details of how we are actually running these commands, can this not be exported?
And maybe we use either "runner" or "executor" for both the type and the struct field to reduce the number of names we use for the same thing.